src/app/hooks/useAuth.tsx

```diff
@@ -15,7 +15,7 @@ const useAuth = () => {
   const onSignIn = useCallback(() => {
     const unsub = onAuthStateChanged(auth, (user) => {
       if (user) {
-        router.push("/course");
+        router.push("/private/course/cs160");
```

Comment:
this is just temporary right? like we would normally just push to /course?

Reply:
Yes, thanks for pointing that out! We will change this once the userContext #9 is merged in.